### PR TITLE
Community survey April 2021

### PR DESCRIPTION
**Pull request by marionf**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Community survey April 2021

### CODE REVIEW

**Review by jolelievre — APPROVED** [review]

**Review by matthieu-rolland — APPROVED** [review]
lgtm !

File: news/_posts/2021-04-14-community-survey.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+The core product team is currently carrying out a survey for the community and oriented around the open-source project.
+
+You are many to express yourself on Slack, on the forum, or on Github about your pains and your wishes about the open-source project.
+
+A lot of issues and suggestions are reported to the project every week. The maintainer’s team's resources are limited, and it’s not possible to integrate all feedbacks experienced by all users.
+
+We want to understand more about your wishes and pains in order to prioritize them.
+
+If you have already contributed to the open-source project, in any way (pull request, bug report, suggestions of new features, participation in translations) then this survey is for you: https://forms-prestashop.typeform.com/to/rHr7tHII
```
Comment (Progi1984) [inline news/_posts/2021-04-14-community-survey.md]:
```suggestion
If you have already contributed to the open-source project, in any way (pull request, bug report, suggestion of new features, participation in translations) then this survey is for you: https://forms-prestashop.typeform.com/to/rHr7tHII
```

File: news/_posts/2021-04-14-community-survey.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+We encourage everyone to express themselves, not only one person by agency or company.
+
+We look forward to your responses, before April 23, in order to be able to establish the roadmap.
+
+We will publish the results of this survey and keep you informed of the roadmap for version 1.7.9.0
```
Comment (Progi1984) [inline news/_posts/2021-04-14-community-survey.md]:
```suggestion
We will publish the results of this survey and keep you informed of the roadmap for version 1.7.9.0.
```

**Review by Progi1984 — COMMENTED** [review]
Small feedbacks

**Review by Progi1984 — APPROVED** [review]

**Review by NeOMakinG — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Pull request merged**